src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,16 @@
+import jwt from 'jsonwebtoken';
+import {SECRET} from '../globals/config';
+
+export function isAuthenticated(req, res, next) {
+  try {
+    let token = req.get('x-token');
+    let decoded = jwt.verify(token, SECRET, {maxAge: 10000});
+    if (decoded) {
+      next();
+    } else {
+      throw new Error();
```

Comment:
I think, it's better to pass `next(error)` or `throw new Error(your message)` then pass it to `catch`